api/shipping_categories.rst

```diff
@@ -0,0 +1,424 @@
+Shipping Categories
+===================
+
+These endpoints will allows you to easily manage shipping categories. Endpoints' base is `/api/shipping-categories`.
+
+Shipping categories have two different serialization groups. While presenting a list you will receive an object with following fields:
+
++-------+-------------------------------------+
+| Field | Description                         |
++=======+=====================================+
+| id    | Id of shipping category             |
++-------+-------------------------------------+
+| name  | Name of shipping category           |
++-------+-------------------------------------+
+| code  | Unique shipping category identifier |
++-------+-------------------------------------+
+
+If you request for a more detailed data, you will receive an object with following fields:
+
++-------------+-------------------------------------+
+| Field       | Description                         |
++=============+=====================================+
+| id          | Id of shipping category             |
++-------------+-------------------------------------+
+| name        | Name of shipping category           |
++-------------+-------------------------------------+
+| code        | Unique shipping category identifier |
++-------------+-------------------------------------+
+| description | Description of shipping category    |
++-------------+-------------------------------------+
+
+.. note::
+
+    Read more about `shipping categories`__ in a components models section.
```

Comment:
"Read more about Shipping Categories" is enough, the second part is useless unless we make it a link but I believe less words is better. :)